src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -83,11 +91,20 @@ and with the largest font setting selected in iOS accessibility settings.
 
 ## Screen readers
 
-Screen readers ([TalkBack][], [VoiceOver][]) enable visually
-impaired users to get spoken feedback about the contents of the screen.
+For mobile, screen readers ([TalkBack][], [VoiceOver][]) enable visually
+impaired users to get spoken feedback about the contents of the screen. Turn on VoiceOver or TalkBack on your mobile device and navigate around your app. 
+
+For web, the following screen readers are currently supported:
+
+Mobile Browsers:
+* iOS - VoiceOver
+* Android - TalkBack
+
+Desktop Browsers:
+* MacOS - VoiceOver
+* Windows - JAWs
 
-Turn on VoiceOver or TalkBack on your device and navigate around your app. If
-you run into any issues, use the [`Semantics` widget][] to customize the
+If you run into any issues, use the [`Semantics` widget][] to customize the
 accessibility experience of your app.
```

Comment:
nit: I'd phrase this differently, something like: "Flutter's standard widgets generate an accessibility tree automatically. However, if your app needs something different, it can be customized using the Semantics widget".